ivy.xml

```diff
@@ -58,7 +58,7 @@
     <dependency org="junit" name="junit" rev="4.8.1" conf="test->default"/>
      <dependency org="org.mockito" name="mockito-all" rev="1.8.2"
                conf="test->default"/>
-    <dependency org="checkstyle" name="checkstyle" rev="5.0"
+    <dependency org="com.puppycrawl.tools" name="checkstyle" rev="6.13"
```

Comment:
Thank you @afine for the patch. I could see checkstyle version "6.13" resulting errors when running with Java 1.6, should we consider java1.6?

`https://zookeeper.apache.org/doc/r3.4.9/zookeeperAdmin.html#sc_supportedPlatforms`

```
/home/max/Projects/zk/git/b-3.4/build.xml:1592: java.lang.UnsupportedClassVersionError: com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask : Unsupported major.minor version 51.0
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClassCond(ClassLoader.java:637)

```